substrate/frame/treasury/src/lib.rs

```diff
@@ -1020,6 +1032,110 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 			// Must never be less than 0 but better be safe.
 			.saturating_sub(T::Currency::minimum_balance())
 	}
+
+	/// Ensure the correctness of the state of this pallet.
+	#[cfg(any(feature = "try-runtime", test))]
+	fn do_try_state() -> Result<(), sp_runtime::TryRuntimeError> {
+		Self::try_state_proposals()?;
+		Self::try_state_spends()?;
+
+		Ok(())
+	}
+
+	/// ### Invariants of proposal storage items
+	///
+	/// 1. `ProposalCount` >= Number of elements in `Proposals`.
```

Comment:
Not sure if linking is of any use here, since the only place where this would show up in the rust-docs is on the pallet struct page (i think), but there the `try_state` doc string is automatically replaced by this: https://github.com/paritytech/polkadot-sdk/blob/5adcb3e10638c8abcfd532b9163990d584fc5832/substrate/frame/support/src/traits/hooks.rs#L374-L383

Reply:
It still is valuable even with no rust docs because it allow using 'go-to-definition' in the comments